File: src/gestorAplicación/externo/Cuotas.java
package gestorAplicación.externo;

import java.util.ArrayList;

public enum Cuotas {
	C1,
	C6,
	C12,
	C18,
	C24,
	C36,
	C48;
	
	public static ArrayList<Cuotas> getCuotas(){
		ArrayList<Cuotas> listaCuotas = new ArrayList<Cuotas>();
		listaCuotas.add(C1);
		listaCuotas.add(C6);
		listaCuotas.add(C12);
		listaCuotas.add(C18);
		listaCuotas.add(C24);
		listaCuotas.add(C36);
		listaCuotas.add(C48);
		return listaCuotas;
	}
}

File: Python/src/gestorAplicación/externo/cuotas.py
from enum import Enum
class Cuotas(Enum):
    #Constantes
    C1 = (1)
    C6 = (6)
    C12 = (12)
    C18 = (18)
    C24 = (24)
    C36 = (36)
    C48 = (48)

    #Constructor
    def __init__(self, cantidad_Cuotas):
        #Atributos de instancia
        self._CANTIDAD_CUOTAS = cantidad_Cuotas

    #Métodos de instancia
    def getCantidad_Cuotas(self):
        return self._CANTIDAD_CUOTAS